blog/2025/04/04-data-delen.md

```diff
@@ -0,0 +1,189 @@
+---
+authors: [martin-van-der-plas]
+tags: [data, api, fds, tfv, oauth,]
+draft: true
```

Comment:
`toc_max_heading_level: 2` graag deze toevoegen. Dan wordt de table of contents aan de rechterkant wat overzichtelijker.